core/dbt/include/global_project/macros/adapters/apply_grants.sql

```diff
@@ -0,0 +1,33 @@
+{% macro get_show_grant_sql(relation) %}
+{{ return(adapter.dispatch('get_show_grant_sql', 'dbt')(relation)) }}
+{% endmacro %}
+
+{% macro default__get_show_grant_sql(relation) %}
+{{ return('show grants on table'(relation.schema)) }}
+{% endmacro %}
+
+{% macro get_grant_sql(relation, grant_config) %}
+{{ return(adapter.dispatch('get_grant_sql', 'dbt')(relation, grant_config)) }}
+{% endmacro %}
+
+{% macro default__get_grant_sql(relation, grant_config) %}
+{{ return(relation) }}
+{% endmacro %}
+
+{% macro get_revoke_sql(relation, grant_config) %}
+{{ return(adapter.dispatch('get_revoke_sql', 'dbt')(relation, grant_config)) }}
+{% endmacro %}
+
+{% macro default__get_revoke_sql(relation, grant_config) %}
+return
+{% endmacro %}
+
+
+{% macro apply_grants(revoke, relation, grant_config) %}
+{{ return(adapter.dispatch('apply_grant', 'dbt')(revoke, relation, grant_config)) }}
+{% endmacro %}
+
+{% macro default__apply_grants(revoke, relation, grant_config) %}
+{{ get_show_grant_sql() }}
+{% return %}
+{% endmacro %}
```

Comment:
If `grant_config` is not specified by the user, `apply_grants` should be a no-op. We should take that to mean that the user does not want dbt running `grant` or `revoke` on their behalf.